ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-rollback-nightly.yaml

```diff
@@ -0,0 +1,221 @@
+base_images:
+  ansible:
+    name: "4.17"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.17"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  hypershift-operator:
+    name: "4.17"
+    namespace: ocp
+    tag: hypershift-operator
+  tests-private-postupg:
```

Comment:
if we don't run upgrade tests, we can remove tests-private-postupg.

Reply:
It's required by hypershift job